src/Analyzer/WebJobs.VSAnalyzer.Vsix/Analyzer2.Vsix.csproj

```diff
@@ -147,5 +147,5 @@
 
 
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
+  <!--<Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />-->
```

Comment:
Is this removal required if you build using the VS Developer command prompt?

Reply:
I didn't try the command prompt, but it was failing in VS. The targets file doesn't exist. Do you expect that to change from the VS command prompt?

Reply:
Not if the SDK is not installed. This essentially introduces the SDK dependency, which you'd need to install, and at that point, building from the dev command prompt should work. I just wonder if removing this will have some impact on the analyzer projects.

Reply:
I installed the SDK and it still failed. It's fair to say I didn't do any explicit testing the analyzer project, just that it built. Is there a flow to test it I can follow?